concent_api/core/tests/test_unit_transfer_operation.py

```diff
@@ -47,31 +53,74 @@ def test_properly_work_of_request_upload_status_function(self):
                 subtask_id                      = '1',
             )
         )
-        file_transfer_token = create_file_transfer_token(
-            report_computed_task,
-            self.REQUESTOR_PUBLIC_KEY,
-            FileTransferToken.Operation.upload,
-        )
 
         with mock.patch('core.transfer_operations.send_request_to_cluster_storage', mock_send_request_to_cluster_correct_response):
-            cluster_response = request_upload_status(
-                file_transfer_token,
-                report_computed_task,
-            )
+            cluster_response = request_upload_status(report_computed_task)
 
         self.assertEqual(cluster_response, True)
 
         with mock.patch('core.transfer_operations.send_request_to_cluster_storage', mock_send_incorrect_request_to_cluster_incorrect_response):
-            cluster_response_2 = request_upload_status(
-                file_transfer_token,
-                report_computed_task,
-            )
+            cluster_response_2 = request_upload_status(report_computed_task)
 
         self.assertEqual(cluster_response_2, False)
 
         with self.assertRaises(UnexpectedResponse):
             with mock.patch('core.transfer_operations.send_request_to_cluster_storage', mock_send_incorrect_request_to_cluster_unexpected_response):
-                request_upload_status(
-                    file_transfer_token,
-                    report_computed_task,
+                request_upload_status(report_computed_task)
+
+
+@override_settings(
+    CONCENT_PRIVATE_KEY=CONCENT_PRIVATE_KEY,
+    CONCENT_PUBLIC_KEY=CONCENT_PUBLIC_KEY,
+)
```

Comment:
If you are using settings e.g `MAXIMUM_DOWNLOAD_TIME` like you doing below you should override them to be sure that they working exactly how you want. Second thing is that if there won't be this setting this test may crash

Reply:
done